sass/_valkey.scss

```diff
@@ -259,7 +259,7 @@ p {
   height: 100%;
   margin: 0 auto;
 
-  .container.right-aside img {
+  .container img {
```

Comment:
`.container img` is pretty broad. Every template uses `.container`, so I'd ideally would prefer it to be a more specific class.

That said, I don't *think* currently we have images that doesn't need to span the container and (eventually) we probably need to work on a better template making this all more/less obsolete, so I think it's fine for now.